src/descriptor/mod.rs

```diff
@@ -619,6 +619,7 @@ impl<Pk: MiniscriptKey + ToPublicKey> Descriptor<Pk> {
     /// transaction. Assumes all signatures are 73 bytes, including push opcode
     /// and sighash suffix. Includes the weight of the VarInts encoding the
     /// scriptSig and witness stack length.
+    /// Panics on malformed Miniscript.
```

Comment:
I think we need to change this to option too. There are miniscripts that have no satisfactions. ex: False, or and(pk, False).

Reply:
It's changed in #152, maybe i should merge the 2 PRs as it also "fixes" (hides?) the compiler test failure.

Reply:
Added in #149